src/ConnectedMode/Resources.resx

```diff
@@ -263,4 +263,13 @@
   <data name="ImportBeforeFileGenerator_WritingTargetFileToDisk" xml:space="preserve">
     <value>[ConnectedMode/ImportsBeforeFileGenerator] Writing file to imports before directory</value>
   </data>
+  <data name="Migration_MigrationPrompt_LearnMoreButton" xml:space="preserve">
+    <value>Learn More..</value>
+  </data>
+  <data name="Migration_MigrationPrompt_Message" xml:space="preserve">
+    <value>SonarLint: We have detected an outdated version of connected mode.</value>
```

Comment:
"We" makes it sound like we are spying on them! ;-)

? `You are using an outdated Connected Mode configuration format. Please migrate to the new format` (though that might be a bit long)

Reply:
I just jused the first half, I feel like the button "migrate" explains the second part good enough